.travis.yml

```diff
@@ -1,2 +1,10 @@
 language: ruby
+rvm:
+- 2.1
 script: "bundle exec jekyll build"
```

Comment:
Travis CI automatically runs that for you when it detects that there's a `Gemfile`: http://docs.travis-ci.com/user/languages/ruby/#Travis-CI-uses-Bundler

You can't use `cache: bundler` if you override the install step: http://docs.travis-ci.com/user/caching/#With-a-custom-install-step, so it's cleaner to leave things as-is and take advantage of the default behavior.